reporter/src/main/kotlin/model/EvaluatedModel.kt

```diff
@@ -138,7 +140,11 @@ data class EvaluatedModel(
             propertyNamingStrategy = PROPERTY_NAMING_STRATEGY
         }
 
-        private val JSON_MAPPER = JsonMapper().apply(MAPPER_CONFIG)
+        private val JSON_MAPPER by lazy {
```

Comment:
Why is this also lazy now (the commit message does not say anything about that), but `YAML_MAPPER` not?

Reply:
Both are made lazy now in a separate commit.